pro/cypress/e2e/migrations/navigation.cy.ts

```diff
@@ -0,0 +1,42 @@
+describe('Navigation', () => {
+  let login = ''
+  const password = 'user@AZERTY123'
+
+  before(() => {
+    cy.visit('/connexion')
+    cy.request({
+      method: 'GET',
+      url: 'http://localhost:5001/sandboxes/pro/create_regular_pro_user',
+    }).then((response) => {
+      login = response.body.user.email
+    })
+  })
+
+  it('I should see the top of the page when changing page', () => {
+    cy.login({
+      email: login,
+      password: password,
+      redirectUrl: '/',
+    })
+    cy.findByText('Bienvenue dans l’espace acteurs culturels')
+    cy.findByText('Vos adresses')
+    cy.findByText('Ajouter un lieu')
+    cy.findAllByTestId('spinner').should('not.exist')
+
+    // I scroll to my venue
```

Comment:
Est-ce qu'il y a une structure créée et sélectionnée: dans le test initial, on avait une étape `And I select offerer "Cinéma du coin"`

Reply:
c'est parce qu'il y aavait plusieurs stuctures maintenant on en a plus qu'une donc pas besoin de la choisir